### ISSUE TITLE
Clean up PingusExit

### ISSUE THREAD

**Issue by Wuzzy2** [the-butterfly-effect/tbe#238]
PingusExit is a bit messy right now, and it can not be used as-is, neither in the scene nor in the toolbox.

PingusExit needs some cleanup so it becomes useful _by default_.

The following things need to be done:

* [ ] Default tooltip (I suggest “Penguins will escape here, never to be seen again.” ;-) )
* [ ] Sane default size (Size used in Pingus level is a good reference)
* [ ] Default image (The door, as in Pingus. But the object would still be open for custom images, I guess)
* [ ] Default name (I suggest “Exit”—short and simple)
* [ ] Clean up existing levels using PingusExit

**Comment by kaa-ching** [comment]
Agreed, it was a bit of a hack. Let's fix it before there are too many of them.
